tuplex/historyserver/thserver/views.py

```diff
@@ -119,6 +106,17 @@ def _jinja2_filter_humanizetime(dt, fmt=None):
 # /api/job              add job with some data
 # /api/
 
+def check_jobs_syntax(jobs):
```

Comment:
great to have a check function here! maybe we can think of some other way to validate the request?

Reply:
This function was actually not needed. I removed it.